output/source-to-season.json

```diff
@@ -1,61 +1,44 @@
 {
   "sources": {
-    "10464158": 12,
     "11666839": 2,
     "13912404": 1,
     "32323943": 11,
     "43842395": 22,
-    "82267399": 16,
     "92433064": 18,
-    "96303009": 1,
     "100617404": 2,
     "110159004": 4,
     "139160732": 15,
     "139599745": 1,
     "148542898": 2,
     "160129377": 18,
     "164083100": 15,
-    "164571094": 1,
-    "186854335": 14,
-    "194661944": 1,
     "210885364": 13,
     "266896577": 5,
-    "277706045": 14,
-    "281362298": 20,
     "286427063": 12,
     "287889699": 13,
-    "288436121": 9,
+    "315474873": 1,
```

Comment:
This is not a seasonally unique source
![image](https://github.com/DestinyItemManager/d2-additional-info/assets/4798491/335a8948-5b1e-41a1-9305-9998fc1fdc63)